Source/WebKit/UIProcess/Network/NetworkProcessProxyCocoa.mm

```diff
@@ -152,6 +156,15 @@
 
     completionHandler(sceneIdentifier, bundleIdentifier);
 }
+
+void NetworkProcessProxy::notifyWillPresentPaymentUI(WebPageProxyIdentifier webPageProxyIdentifier)
+{
+    auto page = WebProcessProxy::webPage(webPageProxyIdentifier);
```

Comment:
in adherence with our safer-cpp guidance (we are explicit about refcounted smart pointer types)
```suggestion
    RefPtr page = WebProcessProxy::webPage(webPageProxyIdentifier);
```